cmake/rocprim-backward-compat.cmake

```diff
@@ -0,0 +1,217 @@
+# ########################################################################
+# Copyright 2021-2022 Advanced Micro Devices, Inc.
+# ########################################################################
+cmake_minimum_required(VERSION 3.16.8)
+
+#Include Directory Paths for - wrapper.
+set(PROJECT_WRAPPER_TEMPLATE_FILE          ${PROJECT_SOURCE_DIR}/header.hpp.in)
+set(PROJECT_WRAPPER_MAIN_INC_DIR           ${PROJECT_BINARY_DIR}/include/rocprim)
+set(PROJECT_WRAPPER_L1_INC_SUBDIR1         ${PROJECT_BINARY_DIR}/include/rocprim/block)
```

Comment:
Instead of SUBDIR1, SUBDIR2 give more appropiate name like detail, device, block....